apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-header/platform-action-bar-header.component.html

```diff
@@ -1,8 +1,33 @@
 <header>Action Bar</header>
 <description>
-    The Action Bar is located at the top of the page and is used for Page title and Main Actions for the page.
+    <p>
+        The Action Bar is located at the top of the page and is used for Page title and Main Actions for the page.

```

Comment:
Documentation does not align with gitbook. There is a reason why we do this:
 - Dont duplicate identical things that are written in core
 - Agree on the content and structure

Reply:
Just read the whole reviews. Its kind of the same but we could write it differently. We have a link there to the core one. 

Since you already returned this back , we would be crazy team to go back forth